homeassistant/components/todoist/calendar.py

```diff
@@ -188,6 +196,27 @@ def handle_new_task(call):
         if _due:
             item.update(due=_due)
 
+        _reminder_due: dict = {}
+        if REMINDER_DATE_STRING in call.data:
+            _reminder_due["string"] = call.data[REMINDER_DATE_STRING]
+
+        if REMINDER_DATE_LANG in call.data:
+            _reminder_due["lang"] = call.data[REMINDER_DATE_LANG]
+
+        if REMINDER_DATE in call.data:
+            due_date = dt.parse_datetime(call.data[REMINDER_DATE])
+            if due_date is None:
+                due = dt.parse_date(call.data[REMINDER_DATE])
+                due_date = datetime(due.year, due.month, due.day)
+            # Format it in the manner Todoist expects
+            due_date = dt.as_utc(due_date)
+            date_format = "%Y-%m-%dT%H:%M:%S"
```

Comment:
I know it's unrelated to your PR, but could you fix the date_format on line 192 above too?

Reply:
Commited.